packages/udes-ui-storybook/storybook/stories/udes-ui-core/index.jsx

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+import Stories from "./stories";
+import { ThemeProvider, theme } from "../../../../udes-ui-core/src";
+
+export default {
+  // eslint-disable-next-line react/display-name, react/prop-types
+  decorator: (storyFn) => (
+    <ThemeProvider theme={theme}>{storyFn()}</ThemeProvider>
```

Comment:
Wrapping the output in this decorator is a nice touch, however a side-effect is that the Prop Types for the `<Button>` components will not be generated under the 'Show info' button in Storybook - could you see if there is a workaround please.

Reply:
Fixed by tweaking some webpack loader settings